tutorials/distributed-calculator/go/Dockerfile

```
@@ -1,8 +1,7 @@
 #first stage - builder
-FROM golang:1.15-buster as builder
+FROM golang:1.19-buster as builder
 WORKDIR /dir
-COPY app.go .
-RUN go get -d -v
+COPY app.go go.mod go.sum ./
 RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .
 #second stage
 FROM debian:buster-slim
```

```suggestion
FROM debian:bookworm-slim
```
Similar idea.  how about we do bookworm instead so we're on latest supported/patched?

---

Note I tested your suggestion and my suggested changes and both work at runtime and are compatible.